packages/tooling/fast-tooling-react/src/style/css-properties.ts

```diff
@@ -0,0 +1,149 @@
+export function assignCSSCustomProperty(
+    customPropertyName: string,
+    customPropertyValue: string | number
```

Comment:
maybe "fallbackValue" or "defaultValue" - that is the purpose this value serves.